docker/app/core/predict.py

```
@@ -0,0 +1,515 @@
+    Returns:
+        翻訳結果のトークンID
+    """
+    if max_length is None:
+        max_length = CONFIG.model_hyperparameters.max_seq_length
+    if input_tensor is None:
+        return None
+
+    # 出力語彙から開始/終了トークンIDを取得（厳密なバリデーション）
+    if '<s>' not in output_vocab:
+        error_msg = (
+            f"出力語彙に '<s>' トークンが存在しません。"
```

Ruff (0.14.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)